packages/channel-provider/README.md

```diff
@@ -0,0 +1,66 @@
+# WalletConnect Channel Provider
+
+Channel Provider for WalletConnect
+
+For more details, read the [documentation](https://docs.walletconnect.org)
+
+## Example
+
+```javascript
+import * as connext from "@connext/client";
+import WalletConnectChannelProvider from "@walletconnect/channel-provider";
+
+/**
+ *  Create WalletConnect Provider (qrcode modal will be displayed automatically)
+ */
+const channelProvider = new WalletConnectChannelProvider({
+  infuraId: "<INSERT_INFURA_APP_ID>"
+});
+
+/**
+ *  Create a channel
+ */
+const channel: Channel = await connext.connect({
+  channelProvider,
+  nodeUrl: "<INSERT_NODE_URL>",
+  store
+});
+
+/**
+ *  Deposit
+ */
+channel.deposit({
+  amount: "0x3abc", // represented as bignumber
+  assetId: AddressZero // Use the AddressZero constant from ethers.js to represent ETH, or enter the token address
```

Comment:
You can enter this as just `amount` as a string, or leave the `assetId` as `undefined` if you are transferring eth.

the `string --> bignumber` and `assetId --> AddressZero` conversions will happen automatically